src/index.css

```diff
@@ -113,3 +113,8 @@ p.MuiTypography-h5.MuiTypography-paragraph {
 .halloween.ph-item .ph-row > div > * {
     background-color: #2b353a;
 }
+
+/* May the Forth Theme */
+.may-the-forth {
+    cursor: url('./app/assets/themes/vulcan.png'), auto;

```

Comment:
This cursor has some jankiness when you're moving around the permanent drawer header. I prefer what you did for the witch (just on the banner).

Also, don't let Kyle see this or he will flip out. Vulcan == Star Trek, not Star Wars

Reply:
oooof, should've looked more carefully next time I check these up

Reply:
> has some jankiness when you're moving around the permanent drawer header.

Sorry what kinds of the "jankiness" are you thinking here? I tried on my end but it seems fine...

Reply:
How is this guy? 

![image](https://user-images.githubusercontent.com/8997218/97311715-0753c680-183b-11eb-9817-2d61c462669e.png)

Reply:
I didn't want to do make it landing-page-only, because I wasn't able to come up with a proper design for the app bar tile (someone knows the culture better can do it). But then the rest of the theme looks just like a regular blue theme. This is why I want to get the to show up everywhere.

Reply:
Jankiness was around the cursor showing up unexpectedly when moving over the drawer header. I'll send you a screen recording to try to illustrate.

Oh, I have some thoughts for the tile: Darth Vader mask, Stormstrooper, AT-AT
![image](https://user-images.githubusercontent.com/29152776/97315547-379d6400-183f-11eb-87c1-b82c21ca6281.png)
![image](https://user-images.githubusercontent.com/29152776/97315601-41bf6280-183f-11eb-84f5-5d50ccde8f73.png)
![image](https://user-images.githubusercontent.com/29152776/97315687-526fd880-183f-11eb-99ee-e3d7b02e7da3.png)